src/07-projects/projects.apib

```diff
@@ -196,17 +196,17 @@ Update a project.
 
     + Attributes (object)
         + id: `dcc2e8ed-51be-4cb6-9c01-034aedac86fd` (string, required)
-        + title: `New company website` (string, required)
-        + description (string, required)
-        + status: `active` (enum, required)
+        + title: `New company website` (string)
+        + description (string)
```

Comment:
Q: isn't this nullable? So when it's null, the description is removed?

Reply:
Not part of the scope, AFAICT.

It shouldn't be too hard to implement it (the Update command seems to already support it), but passing an empty string would work just as well so.. 🤷